pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -400,7 +400,8 @@ func (i *CommandInitOption) RunInit(_ io.Writer, parentCommand string) error {
 
 	// Create karmada kubeconfig
 	serverURL := fmt.Sprintf("https://%s:%v", i.KarmadaAPIServerIP[0].String(), i.KarmadaAPIServerNodePort)
-	if err := utils.WriteKubeConfigFromSpec(serverURL, options.UserName, options.ClusterName, i.KarmadaDataPath, options.KarmadaKubeConfigName,
+	fullUserName := fmt.Sprintf("%s-%s", options.ClusterName, options.UserName)
```

Comment:
Suggest to change global variate directly.

https://github.com/karmada-io/karmada/blob/46dacea8972e061c1b5de108c776b214c7504848/pkg/karmadactl/cmdinit/options/global.go#L16-L21

Suggestion: `ClusterName` is set to `karmada-apiserver`, `UserName` is set to `karmada-admin`

Reply:
thanks, fixed.